include/xrpl/protocol/TxFlags.h

```diff
@@ -189,11 +189,14 @@ constexpr std::uint32_t const tfNFTokenCreateOfferMask =
     ~(tfUniversal | tfSellNFToken);
 
 // NFTokenCancelOffer flags:
-constexpr std::uint32_t const tfNFTokenCancelOfferMask     = ~(tfUniversal);
+constexpr std::uint32_t const tfNFTokenCancelOfferMask     = ~tfUniversal;
 
 // NFTokenAcceptOffer flags:
 constexpr std::uint32_t const tfNFTokenAcceptOfferMask     = ~tfUniversal;
 
+// NFTokenModify flags:
+constexpr std::uint32_t const tfNFTokenModifyMask          = ~tfUniversal;
```

Comment:
You don't need a separate mask - you can also just use the `tfUniversalMask`.

Reply:
fixed